### DISCUSSION TITLE
How to change the website logo

### DISCUSSION THREAD

**Discussion by muhammadkholidb** [colinwilson/lotusdocs#142] [Q&A]
I want to change the logo above the hero section. Is that possible? I see the logo is using an embedded svg in the html. Can I use my own logo to replace it? Thank you!

**Comment by mat-inf** [comment]
I would like that too.
It would be perfect with the ability to set a light and dark-mode version of the logo.

**Comment by colinwilson** [comment]
@muhammadkholidb The next update (v0.2.0) will address this making it definable in the `landing.yaml`

**Comment by drewgrif** [comment]
I am looking through the documentation and do not see logo related content.

**Comment by colinwilson** [comment]
I'm going to update and document this (along with some other PRs) over the coming weekend. Sorry for long delay.

**Comment by rkvsraman** [comment]
Any latest development on this feature?

**Comment by mem-** [comment]
The original question was about change the logo *above* the hero section, which I thing is something different than the "hero" section's logo *at* the landing page.

I would also be interested in changing the logon in the top menu (at the left side, at the same level as the search bar) at regular docs pages and in the upper left corner at the landingpage.

**Reply by mem-** [reply]
I found the answer by looking around inside the example site.

The file **assets/images/logos/mark.svg** is the logo at the landing page
and the file **assets/images/logos/logo.svg** is the logo at search level at docs pages.
